payments/card/other-features.md

```diff
@@ -725,7 +725,7 @@ sequenceDiagram
   note left of Payer: redirect back to merchant<br>(If Redirect scenario)
 
   Payer->>+Merchant: access merchant page
-  Merchant->>+SwedbankPay: GET /psp/creditcard/payments/<paymentorder.id>
+  Merchant->>+SwedbankPay: GET /psp/creditcard/payments/<payment.id>
```

Comment:
```suggestion
  Merchant->>+SwedbankPay: GET /psp/creditcard/payments/<payment-id>
```
`payment.id` is the property where the URL for the payment is stored, I suggest we change it to `-` so when someone copy paste this it won't work.